_tests/browser_tests/README.md

```diff
@@ -0,0 +1,92 @@
+## Quick start:
+
+_Note: If you haven't, make sure to run `./setup.sh from the project root` to install Protractor globally._
+
+From the project root:
+
+```sh
+# Update Selenium Server binaries:
+webdriver-manager update
+```
+
+From this directory:
+
+```sh
+# Start Selenium Server:
+webdriver-manager start
+
+# Open a new tab and tell protractor to start testing:
+protractor conf.js
+```
+
+If you want to test a server other than your local instance (assuming port 7000 is your local CFGov server) use the `--baseUrl` flag. Ex:
+
+```sh
+protractor conf.js --baseUrl=http://beta.consumerfinance.gov
```

Comment:
`environment.js` is for setting environment configurations, however for environments that differ from its defaults it can't be modified without being committed—contrary to its purpose. In other projects I've seen this handled by:
- committing `environments.js` with default values (for the server).
- adding `environments.js` to `.gitignore` and committing that.
- adding and committing `environments.example.js` (with default values) and having instructions in the readme to replace `environments.js` with `environments.example.js`.